### ISSUE TITLE
Nuanced feature requirements

### ISSUE THREAD

**Issue by BD103** [TheBevyFlock/flag-frenzy#15]
`bevy_winit` requires either the `x11` or `wayland` features in order to compile. This is impossible to represent with the current config, because it would force `x11` and `wayland` to always be enabled. We need choices for feature dependencies:

```toml
[features.dependencies]
# `x11` OR `wayland`
bevy_winit = [["x11", "wayland"]]
```

**Comment by BD103** [comment]
`file_watcher` also requires `multi_threaded`, which could be solved by this as well.

**Comment by BD103** [comment]
Closed by #16.
